### DISCUSSION TITLE
游戏后端引擎(skynet等)开发者社区 QQ群 752756240

### DISCUSSION THREAD

**Discussion by coder1777** [cloudwu/skynet#1947] [Show and tell]
大家好，多年来蹭了不少skynet群，感觉大多以应用为主，很少有人讨论后端引擎本身的架构、开发和改进，包括skynet等优秀经久的国产后端引擎。故建此群。

本人自小好学，生性爱才，从不踢人，诚比不上云大cloudwu之才，但从机器码、汇编到C++、Java、Python到形式语言、计算理论、神经网络皆可陪聊。愿有缘者千里相会于网络，经常交流切磋。

QQ群 752756240，PS - 恳请保留此贴。

**Comment by hanxi** [comment]
才2个人？

**Comment by firedtoad** [comment]
How about protected mode os?

涵曦 ***@***.***> 于2024年6月24日周一 09:02写道：
